esignet-service/src/test/java/io/mosip/esignet/controllers/AuthorizationControllerTest.java

```diff
@@ -355,6 +355,287 @@ public void getOauthDetails_withAuthorizeAndOpenIdScope_returnSuccessResponse()
+    }
+
+    @Test
+    public void getOauthDetailsV2_withInvalidResponseType_returnErrorResponse() throws Exception {
+        OAuthDetailRequest oauthDetailRequest = new OAuthDetailRequest();
+        oauthDetailRequest.setClientId("12345");
+        oauthDetailRequest.setRedirectUri("https://localhost:9090/v1/idp");
+        oauthDetailRequest.setScope("openid profile");
+        oauthDetailRequest.setAcrValues("mosip:idp:acr:static-code");
+        oauthDetailRequest.setDisplay("page");
+        oauthDetailRequest.setPrompt("none");
+        oauthDetailRequest.setResponseType("implicit");
+        oauthDetailRequest.setNonce("23424234TY");
+        ZonedDateTime requestTime = ZonedDateTime.now(ZoneOffset.UTC);
+        RequestWrapper wrapper = new RequestWrapper<>();
+        wrapper.setRequestTime(requestTime.format(DateTimeFormatter.ofPattern(UTC_DATETIME_PATTERN)));
+        wrapper.setRequest(oauthDetailRequest);
+
+        mockMvc.perform(post("/authorization/v2/oauth-details")
+                        .content(objectMapper.writeValueAsString(wrapper))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.errors").isNotEmpty())
+                .andExpect(jsonPath("$.errors[0].errorCode").value(ErrorConstants.INVALID_RESPONSE_TYPE));
+    }
+
+    @Test
+    public void getOauthDetailsV2_withOnlyOpenIdScope_returnSuccessResponse() throws Exception {
+        OAuthDetailRequest oauthDetailRequest = new OAuthDetailRequest();
+        oauthDetailRequest.setClientId("12345");
+        oauthDetailRequest.setRedirectUri("https://localhost:9090/v1/idp");
+        oauthDetailRequest.setScope("openid");
+        oauthDetailRequest.setAcrValues("mosip:idp:acr:static-code");
+        oauthDetailRequest.setDisplay("page");
+        oauthDetailRequest.setPrompt("login");
+        oauthDetailRequest.setResponseType("code");
+        oauthDetailRequest.setNonce("23424234TY");
+        ZonedDateTime requestTime = ZonedDateTime.now(ZoneOffset.UTC);
+        RequestWrapper wrapper = new RequestWrapper<>();
+        wrapper.setRequestTime(requestTime.format(DateTimeFormatter.ofPattern(UTC_DATETIME_PATTERN)));
+        wrapper.setRequest(oauthDetailRequest);
+
+        OAuthDetailV2Response oauthDetailV2Response = new OAuthDetailV2Response();
+        oauthDetailV2Response.setTransactionId("qwertyId");
+        when(authorizationService.getOauthDetailsV2(oauthDetailRequest)).thenReturn(oauthDetailV2Response);
+
+        mockMvc.perform(post("/authorization/v2/oauth-details")
+                        .content(objectMapper.writeValueAsString(wrapper))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.response.transactionId").value("qwertyId"));
+    }
+
+    @Test
+    public void getOauthDetailsV2_withOutOpenIdScope_returnSuccessResponse() throws Exception {
```

Comment:
it is returns error, change the method name getOauthDetailsV2_withOutOpenIdScope_returnErrorResponse

Reply:
please check this commit [c4dc9bb](https://github.com/mosip/esignet/pull/265/commits/c4dc9bb091b15804fd39690080c700a5e055404a)